pkgs/tools/security/gnome-keysign/default.nix

```diff
@@ -0,0 +1,54 @@
+{ stdenv, fetchFromGitLab, python3, wrapGAppsHook
+, gobject-introspection
+, gtk3, glib, gnome3, gst_all_1 }:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "gnome-keysign";
+  version = "1.0.1";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.gnome.org";
+    owner = "GNOME";
+    repo = pname;
+    rev = version;
+    sha256 = "0iy70dskd7wly37lpb2ypd9phhyml5j3c7rzajii4f2s7zgb3abg";
+  };
+
+  nativeBuildInputs = [ wrapGAppsHook ] ++ (with python3.pkgs; [
+    lxml Babel
+  ]);
+  buildInputs = [
```

Comment:
Why aren't attributes in `gnome` etc things not on separated newlines?
Helps with merge conflicts.

Reply:
I guess it is legacy thing.